### PR TITLE
[Footer] Create footer

### PR DESCRIPTION
**Pull request by BlueDaisy061**
### Description
- Create footer and styles file
- Add new mixin flex-row
- Add new icons for footer
- Add letter-spacing

### Note: 
Because there is no prototype for this design, so I decided to add each title on Footer as a link to navigate to the other pages.

### Screenshot
footer

### CODE REVIEW

File: src/components/Footer/index.tsx [hunk 1]
```diff
@@ -1,5 +1,110 @@
...
+import styles from './styles.module.scss'
+import { Basketball, BehanceIcon, DiscordIcon, FacebookIcon, InstagramIcon, Logo, TwitterIcon } from '~/assets/icon'
+import { NavLink } from 'react-router-dom'
 
 export const Footer = () => {
-  return <div>Footer</div>
+  return (
+    <div>
+      <div className={styles.footerinfo}>
+        <div className={styles.info}>
```
Comment (tainguy22112000) [inline src/components/Footer/index.tsx]: create component for reusable purpose

File: src/components/Footer/index.tsx [hunk 2]
```diff
@@ -1,5 +1,110 @@
...
+          </div>
+          <div>
+            <p>Process</p>
+          </div>
+        </div>
+        <div className={styles.info}>
+          <div>
+            <NavLink to="/solution" className={styles.title}>Solutions & Services</NavLink>
+          </div>
+          <div>
```
Comment (tainguy22112000) [inline src/components/Footer/index.tsx]: use div for what ?

File: src/components/Footer/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+.footerinfo {
+    padding: 60px 100px;
+    @include flex-row;
+}
+
+.info {
+    div {
+        margin-bottom: 20px;
+        max-width: 278px;
+        line-height: 140%;
```
Comment (tainguy22112000) [inline src/components/Footer/styles.module.scss]: 1.4

File: src/components/Footer/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        p {
+            color: $color-grey;
+        }
+    }
+    .title {
+        color: $color-black;
+        font-weight: bold;
```
Comment (tainguy22112000) [inline src/components/Footer/styles.module.scss]: $font-bold

File: src/components/Footer/index.tsx (hunk 2 above)
Comment (tainguy22112000) [reply src/components/Footer/index.tsx]: use ul li instead

File: src/components/Footer/index.tsx [hunk 5]
```diff
@@ -1,5 +1,110 @@
...
+            <p>Contact</p>
+          </div>
+          <div>
+            <p>Support 24/7: (+84)123456789</p>
+          </div>
+        </div>
+      </div>
+      <div className={styles.moreinfo}>
+        <div className={styles.copyright}>© Copyright 2021-2022</div>
+        <div className={styles.shortenlink}>
```
Comment (tainguy22112000) [inline src/components/Footer/index.tsx]: use ul li instead

File: src/components/Footer/index.tsx (hunk 1 above)
Comment (BlueDaisy061) [reply src/components/Footer/index.tsx]: Updated dude! Thanks

File: src/components/Footer/index.tsx (hunk 2 above)
Comment (BlueDaisy061) [reply src/components/Footer/index.tsx]: No comment for impolite one!

File: src/components/Footer/styles.module.scss (hunk 3 above)
Comment (BlueDaisy061) [reply src/components/Footer/styles.module.scss]: Updated.

File: src/components/Footer/styles.module.scss (hunk 4 above)
Comment (BlueDaisy061) [reply src/components/Footer/styles.module.scss]: My mistake. Updated.

File: src/components/Footer/index.tsx (hunk 5 above)
Comment (BlueDaisy061) [reply src/components/Footer/index.tsx]: Done!

File: src/components/Footer/index.tsx [hunk 6]
```diff
@@ -1,109 +1,23 @@
...
-            <p>Privacy policy</p>
-          </div>
-          <div>
-            <p>Contact</p>
-          </div>
-          <div>
-            <p>Support 24/7: (+84)123456789</p>
-          </div>
-        </div>
+      <div className={styles.container}>
```
Comment (tainguy22112000) [inline src/components/Footer/index.tsx:9]: <ul className={styles.container } >

File: src/components/FooterInfo/index.tsx [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+import { NavLink } from 'react-router-dom';
+
+type props = {
+    title: string | JSX.Element;
+    info: string[];
+    link: string;
+}
+export default function FooterInfo({title, info, link}: props) {
+  return (
+    <div>
```
Comment (tainguy22112000) [inline src/components/FooterInfo/index.tsx]: remove unnecessary div

File: src/components/FooterInfo/index.tsx [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+        <ul>
```
Comment (tainguy22112000) [inline src/components/FooterInfo/index.tsx:13]: <ul> must wrap the outer child components.

File: src/components/Footer/index.tsx (hunk 6 above)
Comment (BlueDaisy061) [reply src/components/Footer/index.tsx:9]: what's wrong with this line bae?

File: src/components/FooterInfo/index.tsx (hunk 7 above)
Comment (BlueDaisy061) [reply src/components/FooterInfo/index.tsx]: Done!

**Pull request merged**